src/utils/cloud.ts

```diff
@@ -165,7 +165,8 @@ export async function downloadOSImage(
 		if (state != null) {
 			return bar.update(state);
 		} else {
-			return spinner.start();
+			spinner.start();
+			return;
 		}
```

Comment:
We can remove the `else` and just call `spinner.start` after. And I think we can remove the `return;`